packages/backend-defaults/src/entrypoints/cache/types.ts

```diff
@@ -17,25 +17,38 @@
 import { LoggerService } from '@backstage/backend-plugin-api';
 import { HumanDuration, durationToMilliseconds } from '@backstage/types';
 import { RedisClusterOptions, KeyvRedisOptions } from '@keyv/redis';
+import { KeyvValkeyOptions } from '@keyv/valkey';
 
 /**
  * Options for Redis cache store.
  *
  * @public
  */
 export type RedisCacheStoreOptions = {
-  type: 'redis' | 'valkey';
+  type: 'redis';
   client?: KeyvRedisOptions;
   cluster?: RedisClusterOptions;
 };
 
+/**
+ * Options for Valkey cache store.
+ *
+ * @public
+ */
+export type ValkeyCacheStoreOptions = {
+  type: 'valkey';
+  client?: KeyvValkeyOptions;
+  cluster?: RedisClusterOptions;
```

Comment:
Is this the literal `RedisClusterOptions`, or is there a `ValkeyClusterOptions` in `@keyv/valkey`?